rancher/en/securing-rancher/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Securing Rancher
+layout: bpg-default-v1.0
+version: v1.0
+lang: en
+---
+
+## Securing Rancher
+---
+
+#### Least Privilege Network Access
+Restricting network access to only required ports/protocols is a good practice for reducing security risk. Rancher only requires a few types of network access:
+
+- Rancher Node to Server: `443/tcp`(ssl) or `8080/tcp`. This is a web socket connection so applying firewalls/IDS that try to inspect HTTP traffic may cause issues.
+- Rancher Node to Node: `500/udp` & `4500/udp`. This is to support `ipsec` connectivity between Rancher
+nodes. **Note**: If you are using `vxlan` the port requirements will be different.
+
+Using cloud providers like AWS, you can restrict traffic between hosts using security groups to only the above hosts.
```

Comment:
This sentence can be more clear with something like:
> When using a cloud provider whose offering includes security policies, you can restrict traffic between hosts to the ports above, plus any additional ports necessary for your workloads to run (such as HTTP/HTTPS or other ports you wish to expose on the host itself).

However, I recommend additional clarity. In AWS, for example, if the security policy for 500/4500 isn't open to `0.0.0.0/0` in certain cases, it won't work. I've seen several scenarios where having it open to only the security group itself fails, but open to the world succeeds. Because those situations exist, and because there is no downside to opening 500/4500/4789 to the world, I recommend that we clarify this to tell them to just open it to the world.